lib/bezier.js

```diff
@@ -727,12 +727,12 @@
       for(i=0; i<len; i++) {
         left = reduced.slice(i,i+1);
         right = reduced.slice(i+2);
-        result = this.curveintersects(left, right);
+        result = this.curveintersects(left, right, null);
```

Comment:
Good point. And, it just worked out that selfintersects should have had the option too.